web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -109,4 +125,17 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
 
   }
 
+   /**
```

Comment:
Coding standards
![image](https://github.com/National-Forestry-Authority/forests/assets/1312689/90669f76-1b33-4677-a974-2f62fdcbc4b8)

Let me know if you need help identifying the warnings.